### PR TITLE
Modified some line

### PR DESCRIPTION
**Pull request by A-Porey**

### CODE REVIEW

File: Chap1_F_d.c [hunk 1]
```diff
@@ -1,11 +1,13 @@
+//program to convert Fahrenheit degrees  temperature into Centigrade degrees.
+
 #include <stdio.h>
 
 int main()
 {
   float C, F;
   printf("Enter the temp. in ferenhite\n");
```
Comment (D-Ghosh) [inline Chap1_F_d.c:8]: Please correct the fahrenheit spelling mistake.

**Comment by D-Ghosh** [comment]
Please correct the fahrenheit spelling mistake in Chap1_F_d.

File: Chap1_F_d.c (hunk 1 above)
Comment (A-Porey) [reply Chap1_F_d.c:8]: Yeah.You are write.

Comment (A-Porey) [reply Chap1_F_d.c]: Done :)

**Comment by D-Ghosh** [comment]
Everything looks good!, great job:)

**Pull request merged**